engine/schema/src/com/cloud/host/dao/HostDao.java

```diff
@@ -72,14 +72,6 @@
 
     List<HostVO> findHypervisorHostInCluster(long clusterId);
 
-    /**
-     * @param type
-     * @param clusterId
-     * @param podId
-     * @param dcId
-     * @param haTag TODO
-     * @return
-     */
```

Comment:
great removal, not all javadoc is helpful ;)

Reply:
yep, I actually do not consider this a documentation ;)